packages/typespec-ts/test/modularUnit/scenarios/models/nullable/nullableUnion.md

```diff
@@ -50,12 +50,6 @@ export type A = {
   message?: string;
   propA?: A;
 } | null;
-/** Alias for A */
-export type A_1 = {
```

Comment:
this is expected as https://github.com/Azure/typespec-azure/issues/1677#issuecomment-2612003284 has been resolved.